cmd/minikube/cmd/start.go

```diff
@@ -549,6 +549,8 @@ func selectDriver(oldConfig *cfg.Config) string {
 		pick, alts := driver.Choose(options)
 		if len(options) > 1 {
 			out.T(out.Sparkle, `Automatically selected the '{{.driver}}' driver (alternates: {{.alternates}})`, out.V{"driver": pick.Name, "alternates": alts})
+		} else {
+			out.T(out.Sparkle, `Automatically selected the '{{.driver}}' driver`, out.V{"driver": pick.Name})
```

Comment:
Maybe put the version of Virsh or Hyperkit.... would help a lot in debugging ?

Reply:
Adding driver versions would be useful, but also a fair bit of work, so I would rather not include it in this already large PR.

Reply:
sounds good.

Reply:
This can also be done as part of hte drivers themselves, as libmachine allows to log these values... this way using the verbose/debug logs for this would collect them. This is also what we do for CRC.